pkg/login/login.go

```diff
@@ -86,7 +93,9 @@ func (c *Client) challengeAuth() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	if resp.StatusCode != 401 {
+	if resp.StatusCode == 200 {
+		return "", ErrAuthDisabled
+	} else if resp.StatusCode != 401 {
```

Comment:
This is a very small nit, but why using else when the previous statement returns? As the previous `if err != nil` condition returns too, it'd be more consistent to write : 
``` go
if err != nil {
    return "", err
} else if resp.StatusCode == 200 {
    return "", ErrAuthDisabled
} else if resp.StatusCode != 401 {
    return "", fmt.Errorf("expected status code 401, got %d", resp.StatusCode)
}
```

I think it's clearer to avoid "else / else if" altogether when the previous block returns, though the benefit is more visible for else blocks (as there is then no extra indentation level).